tls/s2n_tls13.h

```diff
@@ -24,20 +24,19 @@ extern "C" {
 #endif
 
 #if S2N_GCC_VERSION_AT_LEAST(4, 5, 0)
-    S2N_API
-    __attribute__((deprecated("The use of TLS1.3 is configured through security policies")))
-    extern int s2n_enable_tls13();
+S2N_API
```

Comment:
This one probably will have to be fixed manually. Looks like extern int should be on the same line as s2n_enable_tls13(). Not really sure what's going on with the S2N_API macro...? I think it should be indented and on the same line as __attribute__?

Reply:
Yeah we added `S2N_API` to clang-format's `AttributeMacros`. That forced us to but S2N_API on the same line as the rest of the declaration. If do the same thing here the whole thing becomes one line and clang-format is happy.